3/test/excercise3.spec.js

```diff
@@ -0,0 +1,110 @@
+const { gql } = require('apollo-server')
+const server = require('../server.js')
+const {createTestClient} = require('apollo-server-testing');
+const jwt = require('jsonwebtoken');
+
+const { query, mutate } = createTestClient(server);
+
+describe('Query: Given a list of todos and assignees', () => {
+
+    const queryTodoLength = gql`query getCountTodos {countTodos}`;
+    const queryAssigneeLength = gql`query getCountAssignees {countAssignees}`;
+    
+    it('get all todo items at startup', () => {
+        query({query: queryTodoLength}).then((result) => {
+            expect(result.data.countTodos).toBe(6);
+        })
+    })
+    it('get all assignees at startup', () => {
+        query({query: queryAssigneeLength}).then((result) => {
+            expect(result.data.countAssignees).toBe(3);
+        })
+    })
+
+    const querySignedAssignee = gql`query getSigneAssignee($token: String!) { assignees(token: $token) {id, name} }`;   
+    const tokenMessage = jwt.sign({auth: 'der richtige inhalt'}, 'shhhhh');
+
+    it('get all assignees signed', () => {
+        query({query: querySignedAssignee, variables: {token: tokenMessage}}).then((result) => {
+            expect(result.data.assignees.length).toBe(3);
+        })
+    })
+})
+
+describe('Query: Given todos with different states', () => {
+
+    const queryTodosByState = gql`query getTodosByState($state: State!) { todosByState(state: $state) {id, message} }`;
+
+    it('return done todos', () => {
```

Comment:
```suggestion
    it('returns done todos', () => {
```
You should write your test case like a sentence in simple present:
> it returns done todos

Or maybe better:
> it returns finished todos

```suggestion
    it('returns finished todos', () => {
```